modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/RoleResourceTest.java

```diff
@@ -332,58 +462,61 @@ protected Role testGraphQLRole_addRole() throws Exception {
 	protected Role testPostOrganizationRoleUserAccountAssociation_addRole()
 		throws Exception {
 
-		return _addRole(RoleConstants.TYPE_ORGANIZATION);
+		return _addRole(true, RoleConstants.TYPE_ORGANIZATION);
 	}
 
 	@Override
 	protected Role testPostRole_addRole(Role role) throws Exception {
 		role.setRoleType(
 			RoleConstants.getTypeLabel(RoleConstants.TYPE_REGULAR));
 
-		return _addRole(role);
+		return _addRole(true, role);
 	}
 
 	@Override
-	protected Role testPostRoleUserAccountAssociation_addRole()
+	protected Role
+			testPostRoleByExternalReferenceCodeUserAccountAssociation_addRole()
 		throws Exception {
 
-		return _addRole(RoleConstants.TYPE_REGULAR);
+		return _addRole(false, RoleConstants.TYPE_REGULAR);
+	}
+
+	protected Long
```

Comment:
missing override

Reply:
There is no such method in Base class. I could rename it to avoid confusion,